Launcher/Classes/Extensions.swift

```diff
@@ -62,3 +62,18 @@ extension Dictionary where Key == String {
         }
     }
 }
+
+extension Dictionary where Value: Equatable {
```

Comment:
I am curious about this. Dictionary keys must be unique, so why not just `dictionary[key]`? This never returns more than 1 result, does it?

Reply:
This is key for value. There are multiple translation keys for values like "Messages"